src/app.module.ts

```diff
@@ -18,9 +23,22 @@ import { ConsoleModule } from './console/console.module';
       },
       resolvers: { JSON: GraphQLJSON },
     }),
+
     ContributionManagerModule,
     ConsoleModule,
     EventEmitterModule.forRoot(),
+    AuthModule,
+    UsersModule,
+  ],
+  providers: [
+    {
+      provide: APP_GUARD,
+      useClass: AuthGuard,
+    },
+    {
+      provide: APP_GUARD,
+      useClass: RolesGuard,
+    },
```

Comment:
i think this duplicates `@Injectable` annotations

Reply:
I wanted to put this  decorators globally (even with the graphQl requests). Dev will have to explicitly add the @Public() deocrator if they want their api to be public.